src/CommonUtilities/Base32.cs

```diff
@@ -64,6 +64,7 @@ public static class Base32
             => ts
                 .Select((value, index) => (Index: index, Value: value))
                 .GroupBy(tuple => tuple.Index / itemsPerGroup)
+                .OrderBy(tuple => tuple.Key)
```

Comment:
Is this fixing a bug?  Just want to be sure this was intentionally included in the PR